src-cpp/rdkafkacpp.h

```diff
@@ -1867,6 +1871,7 @@ class RD_EXPORT Headers {
     }
 
  private:
+    /** @remark: expects initialized value argument. */
```

Comment:
I think that is implicit (and true for most function arguments), so no need to comment it here.